proto/osmosis/gamm/v1beta1/query.proto

```diff
@@ -106,15 +106,15 @@ message QueryTotalSharesResponse {
 
 //=============================== PoolAssets
 message QueryPoolAssetsRequest {
-  uint64 poolId = 1 [ (gogoproto.moretags) = "yaml:\"pool_id\"" ];
+  uint64 poolID = 1 [ (gogoproto.moretags) = "yaml:\"pool_id\"" ];
 }
 message QueryPoolAssetsResponse {
   repeated PoolAsset poolAssets = 1 [ (gogoproto.nullable) = false ];
 }
 
 //=============================== SpotPrice
 message QuerySpotPriceRequest {
-  uint64 poolId = 1 [ (gogoproto.moretags) = "yaml:\"pool_id\"" ];
+  uint64 poolID = 1 [ (gogoproto.moretags) = "yaml:\"pool_id\"" ];
```

Comment:
Is it straightforward for us to delay the Id -> ID change to another PR? I'm worried its going to cause a breaking change that we're not going to easily catch (And makes a bunch of merge conflicts with other things)

Reply:
yep we can do that.  I think id just need to change a setting in revive.

Reply:
actually, I should rephrase-- it isn't striaghtforward at all, est maybe a day's work but I already think that I need to dial back to this commit:

https://github.com/osmosis-labs/osmosis/pull/1036/commits/041617c503fe9f627b6f32bbb64dede751becd8f